src/models/forecasts.jl

```diff
@@ -411,3 +411,18 @@ function make_forecasts(forecast::Forecast, interval::Dates.Period, horizon::Int
 
     return forecasts
 end
+
+"""
+    make_forecasts(forecast::FlattenIteratorWrapper{T}, 
+                    interval::Dates.Period, horizon::Int) where T <: Forecast
+                    
+Make a vector of forecasts by incrementing through a vector of forecasts 
+by interval and horizon.
+"""
+function make_forecasts(forecast::FlattenIteratorWrapper{T}, 
+                        interval::Dates.Period, horizon::Int) where T <: Forecast
+
+    forecasts = [make_forecasts(f, interval, horizon) for f in forecast]
+
+    return vcat(forecasts...)
```

Comment:
Does this need to be a vector?  You could avoid extra memory allocations if you used FlattenIteratorWrapper.